### DISCUSSION TITLE
Preparing for new CAM tag based on coupled run 51

### DISCUSSION THREAD

**Discussion by cecilehannay** [NCAR/amwg_dev#412] [X: General]
As discussed at the last cam7 meeting, we would like to create a new cam tag to bring the head of cam closer to the simulations we are doing at the moment. The ocean group is planning to do the same. Then, CSEG will create a new cesm tag for the coupled simulations. 

**The purpose of this thread is to collect info to create the new tag.** 

Here are the latest coupled runs:
![Screen Shot 2023-10-09 at 1 32 49 PM](https://github.com/NCAR/amwg_dev/assets/9723220/dc0a4291-955d-41b8-98f9-67caa5d7cd07)

As discussed at the cam7 meeting, the current idea is to use the SourceMods and namelist mods from 51. 

**Namelist mods:**
```
clubb_gamma_coef = 0.3
clubb_gamma_coefb = 0.3

micro_mg_dcs=500.D-6
micro_mg_vtrmi_factor = 1.0
microp_aero_wsubi_scale  = 1.0
microp_aero_wsub_scale  = 0.25
microp_aero_wsub_min   =   0.
```

**SourceMods:**
- changes to ice clouds (from Toniazzo)
```
/glade/p/cesmdata/cseg/runs/cesm2_0/b.e23_alpha16b.BLT1850.ne30_t232.051/SourceMods/src.cam/cldfrc2m.F90 
```

- floor value for wsub is also scaled by 0.5 (10->5 cm/s)
```
/glade/p/cesmdata/cseg/runs/cesm2_0/b.e23_alpha16b.BLT1850.ne30_t232.051/SourceMods/src.cam/ndrop.F90 
```
My recollection is that this mod is dependent of `microp_aero_wsub_min`. 
Adam: we have been dragging this for a while. Please confirm we still want this. 

**Contacts:**
@PeterHjortLauritzen, @adamrher, @JulioTBacmeister, @swrneale, @tto061
@cacraigucar, @jedwards4b

**Comment by PeterHjortLauritzen** [comment]
Thank you @cecilehannay!

For those who want to follow the details on the code mods please see https://github.com/ESCOMP/CAM/issues/895

**Reply by cecilehannay** [reply]
Thanks @PeterHjortLauritzen: https://github.com/ESCOMP/CAM/issues/895 is **"The One"**

Note that there were 2 open issues but I closed https://github.com/ESCOMP/CAM/issues/899 as it was redundant with https://github.com/ESCOMP/CAM/issues/895.

**Comment by cacraigucar** [comment]
Several questions:
1. I assume that the namelist changes should only apply to `phys="cam_dev"` runs and not for all versions of CAM in cam_development?
2. In ndrop, wmixmin is being changed.  Is this a one-time change, or should it become a namelist?
3. In cldfrc2m, qist_min is being changed.  Is this a one-time change, or should it become a namelist?

**Reply by adamrher** [reply]
Re: 1. Yes.
Re: 2. I don't think we need to change this hard coded value as it only really matters for extremely small values of the wscale_factor namelists (like 0.01), and I am not comfortable and don't think we'll be using extremely small values of wscale_factor. @JulioTBacmeister?
Re: 3. I like the idea of making this a namelist.

@cacraigucar per our discussion of the changes to cldfrc2m at the 9AM mtg today, how about I go ahead and make the changes to cldfrc2m using the latest tag, and then I could point you to it and you could copy it into your PR branch (and make the required namelist_defaults/defintions/build-namelist changes I introduce in cldfrc2m)?

**Reply by adamrher** [reply]
edit - actually Ill make the namelist changes as well since I want to make sure it can build and run.

**Comment by cacraigucar** [comment]
So as of right now (1:45 on Oct 10), I have:

- The namelist changes as provided by Cecile
- A proposal that ndrop is NOT being changed (@JulioTBacmeister - This question is directed to you from Adam)
- An updated cldfrc2m coming from Adam
